srcalc/src/main/webapp/WEB-INF/views/fragments/multiSelectFragment.jsp

```diff
@@ -0,0 +1,20 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+
+<c:choose>
+    <c:when test="${displayItemParam.displayType == 'Radio'}">
```

Comment:
Simplification: you could have `MultiSelectVariableView.getFragmentName()` alternate based on the display type. This `DisplayItem` hierarchy provides good polymorphism!